client/templates/Report/Formatted/Accounts/_transfers.html.twig

```diff
@@ -5,14 +5,15 @@
     <h2 class="section-heading">Money transfers</h2>
     <div class="box box-heading flush--bottom">
         <h3 class="flush--bottom">Money transfers</h3>
-    </div>
```

Comment:
Looks like there's a deleted character here. Not sure if it actually affects the final look, but worth just fixing it anyway.

Reply:
Yeah, improperly closed tags can do strange things to layout, as it throws the nesting. If we haven't already, worth building in a validator check to our workflow/process to catch issues lie that.